cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/UpdateTopicConfigurationRunnable.java

```diff
@@ -209,8 +209,8 @@ protected OptimizerResult workWithClusterModel() throws KafkaCruiseControlExcept
       _kafkaCruiseControl.executeProposals(result.goalProposals(), Collections.emptySet(), false, _concurrentInterBrokerPartitionMovements,
           _maxInterBrokerPartitionMovements,
           0, _clusterLeaderMovementConcurrency, _brokerLeaderMovementConcurrency,
-          _executionProgressCheckIntervalMs, _replicaMovementStrategy, _replicationThrottle, _isTriggeredByUserRequest, _uuid,
-          SKIP_AUTO_REFRESHING_CONCURRENCY);
+          _executionProgressCheckIntervalMs, _replicaMovementStrategy, _replicationThrottle, null,
```

Comment:
The `logDirThrottle` is null here because partitions are never any intra-broker partition movements as part of topic configuration changes, is my understanding correct?